docs/spfx/publish-to-marketplace-checklist.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Prepare your SharePoint Framework application for publishing to the marketplace
+description: Tips & tricks to help you get your SharePoint Framework application published in the marketplace
+ms.date: 06/25/2020
+ms.prod: sharepoint
+localization_priority: Priority
+---
+
+# App validation checklist
```

Comment:
Should match the document title... the H1 value is what is shown in the page itself. The `title` metadata property is used in the <HTML><HEAD></></> section.